packages/components/stories/ActionBar.js

```diff
@@ -249,11 +249,12 @@ storiesOf('ActionBar', module)
 					selected={1}
 					actions={{ left: [primary] }}
 					multiSelectActions={{ right: [multi3] }}
+					getComponent="whatever"
 				/>
 			</div>
 			<p>Toolbar with btn-group and only icons/ Layout: left, center, right</p>
 			<div id="btn-group">
-				<ActionBar actions={btnGroupActions} />
+				<ActionBar actions={btnGroupActions} getComponent="whatever" />
```

Comment:
`ActionBar.propTypes` mentions `func` for this prop, shouldn't it be updated or the story changed then ?

Reply:
Yep I was wondering about this prop too, it's not really used as we inject just a known React component. But it's actually necessary in the `injectAll` mechanic for now (no injection if no `getComponent`).
I will change here the story with an empty function to match the proptypes. And later I will make the correction in the injection mechanic in an other PR.

Reply:
If it's a component, maybe `PropTypes.node` then :+1: